Minor ajdustments

jboisseur opened the pull request:
Hi multicolor,
This is zacharielle from Bludit forum (https://forum.bludit.org/viewtopic.php?t=2449) ;)
Thanks again for this plugin. 
If you agree, I suggest adding a French language json file as well as deleting the utf8_decode function in plugin.php so to allow accentuated characters within the password. It works fine with my instance but feel free to reject the changes if they don't suit you.
All the best!

jboisseur committed:
Adding French

jboisseur committed:
Removed utf8_decode function on line 26 to manage accented char

multicolor-rgb commented:
ok, i will test it and reply

multicolor-rgb commented:
Hi!
I am done:)  - This is version 2.0
Download:
https://github.com/multicolor-rgb/premium-post/archive/refs/heads/2.0.zip

you can check:
What i changed?
-moved security folder to Bl-content
-remove extension .txt
-removed utf8 encoding.

Please give me feedback, and if you like my plugins you can buy me coffe:)
https://bit.ly/donate-multicolor-plugins

jboisseur commented:
Hi multicolor,

Sorry it took me so long to check this out. Thank you for this new version! 

I see it is already available on Bludit site :O

Here are my remarks:
-> about the changes you mention:
-moved security folder to Bl-content: as long as it's deleted / deactivated when the plugin is deleted / deactivated it sounds fine to me
-remove extension .txt: no change on user side so I think this is OK :)
-removed utf8 encoding: works fine now, thanks!

-> about other things I noticed
-thanks for adding the french json language
-in plugin.php, maybe you should modifiy function form() so to explain a bit more what's to be done to use the plugin:
. add "in bl-themes/$theme_name/php/page.php" prior to "replace"
. change "<?php echo $page->content() ;?> ": by "<?php echo $page->content(); ?>)" (there's a space typo that prevents copy/paste from working)
. maybe change "to" with "by"?
. maybe cou could suggest also suggest users to change "<?php echo $page->contentBreak(); ?>" in home.php in theme for passwording articles?

Thanks!
